index.html

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 
+
 <head>
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta property="og:title" content="To-Do-Manager">
+  <meta property="og:description" content="To-Do-Manager is a web app used to maintain our day-to-day tasks or list everything that we have to do. This project provides an easy way to add, edit and delete tasks. Added tasks will be stored in the browser’s local storage so, they won’t be removed on page refresh or tab close.">
+  <meta property="og:image" content="./assets/images/to-do icon.png">
+  <meta property="og:url" content="https://malivinayak.github.io/To-Do-Manager">
   <title>To-Do-Manager</title>
```

Comment:
icon is not added
previously you have added icon but after your latest commit icon is not present
kindly verify it and make necessary changes

Reply:
Oh, is that so? Alright, let me do a final check and make the fixes as requested.
@malivinayak